cli/command/cluster/create.go

```diff
@@ -29,7 +29,7 @@ func NewCreateCommand(c cli.Interface) *cobra.Command {
 	flags.BoolVarP(&opts.notifications, "notifications", "n", false, "Enable/disable server notifications (default is 'false')")
 	//flags.StringVar(&opts.provider, "provider", "local", "Cluster provider (\"local\" (default) or \"aws\")")
 	flags.StringVarP(&opts.registration, "registration", "r", configuration.RegistrationNone, "Specify the registration policy (possible values are 'none' or 'email')")
-	flags.StringVarP(&opts.tag, "tag", "t", c.Version(), "Specify tag for cluster images (use 'local' for development)")
+	flags.StringVarP(&opts.tag, "tag", "t", c.Version(), "Specify tag for cluster images (use '0.13.0-dev' for development)")
```

Comment:
`(use '0.13.0-dev' for development)` I'm not sure we need to document that inside the CLI. This should be part of the dev documentation instead.